### PR TITLE
49774 - manually run pos webjob to create exchange set

### PR DESCRIPTION
**Pull request by sumod13943**
Azure table elements added.
Entries added to an Azure table to track the execution of webjob.
Changes in pipeline scripts

### CODE REVIEW

**Review by ray-sj — CHANGES_REQUESTED** [review]
Where are the steps for executing this task

**Review by ajasonca — APPROVED** [review]
Looks good

File: UKHO.PeriodicOutputService/UKHO.PeriodicOutputService.Fulfilment.UnitTests/Handler/FakeHttpMessageHandler.cs [hunk 1]
```diff
@@ -5,14 +5,14 @@ namespace UKHO.PeriodicOutputService.Fulfilment.UnitTests.Handler
...
-        private HttpResponseMessage _response;
+        private readonly HttpResponseMessage _response;
 
         public static HttpMessageHandler GetHttpMessageHandler(string content, HttpStatusCode httpStatusCode)
         {
             var response = new HttpResponseMessage()
             {
                 StatusCode = httpStatusCode,
-                Content = new StringContent(content, Encoding.UTF8, "application/json")
+                Content = new StringContent(content, Encoding.UTF8, "application/json"),
```
Comment (harshal11869) [inline UKHO.PeriodicOutputService/UKHO.PeriodicOutputService.Fulfilment.UnitTests/Handler/FakeHttpMessageHandler.cs:15]:
```suggestion
                Content = new StringContent(content, Encoding.UTF8, "application/json")
```

File: UKHO.PeriodicOutputService/UKHO.PeriodicOutputService.Common/Helpers/AzureTableStorageHelper.cs [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        private readonly IOptions<AzureStorageConfiguration> _azureStorageConfig;
+
+        public AzureTableStorageHelper(IOptions<AzureStorageConfiguration> azureStorageConfig)
+        {
+            _azureStorageConfig = azureStorageConfig;
+        }
+
+        public void SaveHistory(WebJobHistory webJobHistory)
+        {
+            TableClient tableWebjobEntityClient = GetTableClient(WEBJOB_HISTORY_TABLE_NAME);
```
Comment (harshal11869) [inline UKHO.PeriodicOutputService/UKHO.PeriodicOutputService.Common/Helpers/AzureTableStorageHelper.cs:23]:
```suggestion
            TableClient tableWebJobEntityClient = GetTableClient(WEBJOB_HISTORY_TABLE_NAME);
```

**Review by harshal11869 — CHANGES_REQUESTED** [review]
Minor correction

File: UKHO.PeriodicOutputService/UKHO.PeriodicOutputService.Common/Helpers/AzureTableStorageHelper.cs (hunk 2 above)
Comment (sumod13943) [reply UKHO.PeriodicOutputService/UKHO.PeriodicOutputService.Common/Helpers/AzureTableStorageHelper.cs]: Changed

**Review by harshal11869 — APPROVED** [review]
LGTM

**Review by mattwin1 — APPROVED** [review]
Devops changes LGTM

**Pull request merged**